rust/ql/src/queries/security/CWE-327/BrokenCryptoAlgorithm.qhelp

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE qhelp PUBLIC
+"-//Semmle//qhelp//EN"
+"qhelp.dtd">
+<qhelp>
+     <overview>
+          <p>
+               Using broken or weak cryptographic algorithms can leave data
+               vulnerable to being decrypted or forged by an attacker.
+          </p>
+
+          <p>
+               Many cryptographic algorithms provided by cryptography
+               libraries are known to be weak, or flawed. Using such an
+               algorithm means that encrypted or hashed data is less
+               secure than it appears to be.
+          </p>
+
+          <p>
+               This query alerts on any use of a weak cryptographic algorithm, that is
+               not a hashing algorithm. Use of broken or weak cryptographic hash
+               functions are handled by the
+               <code>rust/weak-sensitive-data-hashing</code> query.
```

Comment:
This query doesn't exist yet, but I guess you are planning to add it shortly, so it's fine to leave it like this,

Reply:
Also a good point.  We do indeed plan to add it soon, but I'll make a note to modify this if we do not.